From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct loop_device
Patch-mainline: never, kabi
References: kabi

In 4.4.141, commit 43991e916eb9eac9ea0ef0317f9bfd3c491972cb (loop:
remember whether sysfs_create_group() was done), upstream commit
d3349b6b3c373ac1fbfb040b810fcee5e2adc7e0 added one bool struct
loop_device. It made the kABI checker to complain.

Given it added a bool to a hole, just hide the change from the kABI
checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/block/loop.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/block/loop.h
+++ b/drivers/block/loop.h
@@ -60,7 +60,9 @@ struct loop_device {
 	struct kthread_worker	worker;
 	struct task_struct	*worker_task;
 	bool			use_dio;
+#ifndef __GENKSYMS__
 	bool			sysfs_inited;
+#endif
 
 	struct request_queue	*lo_queue;
 	struct blk_mq_tag_set	tag_set;
